app/models/persistent_volume.rb

```diff
@@ -1,5 +1,6 @@
 class PersistentVolume < ContainerVolume
   acts_as_miq_taggable
+  include NewWithTypeStiMixin
```

Comment:
Oh I just noticed that this should be on the base class, not here in the subclass.